lib/galaxy/tools/test.py

```diff
@@ -157,7 +157,8 @@ def __process_raw_inputs( self, tool_inputs, raw_inputs, parent_context=None ):
                     for input_name, input_value in case.inputs.items():
                         case_inputs = self.__process_raw_inputs( { input_name: input_value }, raw_inputs, parent_context=cond_context )
                         expanded_inputs.update( case_inputs )
-                    expanded_case_value = self.__split_if_str( case.value )
+                        if not value.type == "text":
```

Comment:
I think you want to unindent this one level right? (Or did the original code have a problem?)